package.json

```diff
@@ -15,11 +15,15 @@
   "packageManager": "yarn@4.2.2",
   "resolutions": {
     "ansi-regex": "5.0.1",
+    "cheerio": "1.0.0-rc.12",
     "diff": "3.5.0",
     "glob-parent": "5.1.2",
     "json-schema": "0.4.0",
+    "minimatch": "3.1.2",
```

Comment:
Minimatch was upgraded to 3.1.2 instead of 3.0.5 because it's the latest 3.x version that maintains CommonJS API compatibility with karma while addressing all ReDoS vulnerabilities.